wolfssl/wolfcrypt/settings.h

```diff
@@ -1391,11 +1391,16 @@ extern void uITRON4_free(void *p) ;
 #ifdef MICRIUM
     #include <stdlib.h>
     #include <os.h>
-    #include <net_cfg.h>
-    #include <net_sock.h>
-    #include <net_err.h>
+    #if defined(RTOS_MODULE_NET_AVAIL) || (APP_CFG_TCPIP_EN == DEF_ENABLED)
+        #include <net_cfg.h>
+        #include <net_sock.h>
+        //#include <net_err.h>
```

Comment:
Will commenting out `net_err.h` possibly cause backwards compat issues? Also please avoid `//`.

Reply:
Corrected for different versions of Micrium now.